src/main/resources/archetype-resources/core/src/main/java/internal/models/HierarchyPageImpl.java

```diff
@@ -23,8 +23,8 @@
 import java.util.Map;
 import java.util.regex.Pattern;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import org.jetbrains.annotations.NotNull;
```

Comment:
[Minor] - Import statement order needs to be fixed.
Ideally, the order of package imports is -

java.*
_blank line_
javax.*
_blank line_
org.*
_blank line_
*
_blank line_
import static *